features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/identity.xml.j2

```diff
@@ -2724,6 +2724,7 @@
         </Resource>
         <Resource context="(.*)/filedownload(.*)" secured="false" http-method="all"/>
         <Resource context="(.*)/logincontext(.*)" secured="false" http-method="all"/>
+        <Resource context="(.*)/longwaitstatus(.*)" secured="false" http-method="all"/>
```

Comment:
We just need POST right?

Reply:
We need to GET.. do u want to fine-grain the HTTP-method?

Reply:
Fine grained here -  tested https://github.com/wso2/carbon-identity-framework/pull/4502/commits/a903ed746f19fadb9a54d27c511f2f030bac0eb2